pallets/asset-manager/src/lib.rs

```diff
@@ -396,6 +433,54 @@ pub mod pallet {
 		}
 	}
 
+	impl<T: Config> Contains<MultiLocation> for Pallet<T> {
+		fn contains(location: &MultiLocation) -> bool {
```

Comment:
Can you write Rust doc describe what does this method do?

Reply:
Updated.

Reply:
`/// Check the multilocation is supported by calamari/manta as a destination chain`